### DISCUSSION TITLE
[Feature] Share photos with exact person / Album Autocreation

### DISCUSSION THREAD

**Discussion by nett00n** [immich-app/immich#3895] [Feature Request]
### The feature

I want to share every photo, my friend Alice is presented on to Alice.

It can be autocreated album with rule "any photo with Alice added to this album" or a "share" feature is added to "people" page

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of  #3306
